fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -35,6 +35,7 @@
 #include "flow/actorcompiler.h" // This must be the last #include.
 
 #define OP_DISK_OVERHEAD (sizeof(OpHeader) + 1)
+#define ENCRYPTION_ENABLED_BIT 31
```

Comment:
I'm a bit paranoid, but let's `static_assert(sizeof(int) == 32)`

Reply:
Agree, 16 bit architecture it would be 2 bytes, but, for all others it should be 4 bytes. Anyhow, given the rigid KVStore disk layout constraints at the moment, I agree being paranoid is good :).

Reply:
I can add it but it will be `== 8` since `sizeof` its in bytes not bits